librz/core/cmd_descs/cmd_print.yaml

```diff
@@ -1447,6 +1447,25 @@ commands:
           - name: skip
             type: RZ_CMD_ARG_TYPE_RZNUM
             optional: true
+      - name: p=r
+        summary: Print rising and falling entropy by default rising threshold is 0.95 and falling threshold is 0.85
```

Comment:
The information about default threshold values is probably belongs in `details` not `summary`

Reply:
Okay, I will change that.